test/cqlpy/test_materialized_view.py

```diff
@@ -1409,7 +1408,7 @@ def test_view_forbids_table_ops(cql, mv1):
     # that DESC TABLE cannot be used on a view and that DESC MATERIALIZED VIEW
     # should be used instead - it just reports that the table is "not found".
     # Reproduces #21026 (DESC TABLE was allowed on a view):
-    with pytest.raises(InvalidRequest):
+    with pytest.raises(InvalidRequest, match='Cannot use'):
```

Comment:
This test will fail on Cassandra, because if the comment above is correct (which I wrote, so I hope it is), Cassandra just reports that the table is not found - and the message does not include "Cannot use".
I am NOT saying that we need to be compatible with Cassandra in the error message, and your approach (to do what DROP TABLE et al. do) is better, but still, the test needs to pass on both Cassandra and Scylla. So you should either use a regex with "or" of two different messages, or should have just left the code I wrote in the test, checking just the type and not the message.

Reply:
I already queued this PR, so this will have to be done in a follow-up. @Jager227 please follow-up with a fix for the above request.